WordPress/Classes/ViewRelated/Jetpack/Branding/Banner/JetpackBannerWrapperViewController.swift

```diff
@@ -10,6 +10,11 @@ final class JetpackBannerWrapperViewController: UIViewController {
     /// JPScrollViewDelegate conformance.
     internal var scrollViewTranslationPublisher = PassthroughSubject<Bool, Never>()
 
+    override var navigationItem: UINavigationItem {
+        guard let childVC else { return UINavigationItem() }
```

Comment:
```suggestion
        guard let childVC else { return super.navigationItem }
```
No concrete reason tbh. But it seems cleaner/safer.

Reply:
Good call - tweaked in ab695b715a3275ee3600dd1b3ea38946be6c4816.